cli/src/main/scala/com/codacy/analysis/cli/command/CLIApp.scala

```diff
@@ -146,6 +146,8 @@ final case class Analyze(
   skipCommitUuidValidation: Int @@ Counter = Tag.of(0),
   @ExtraName("u") @ValueDescription("If the results should be uploaded to the API")
   upload: Int @@ Counter = Tag.of(0),
+  @ExtraName("i") @ValueDescription("[default: false] - Skips ssl verification when comunicating with Codacy API")
```

Comment:
We also need to apply the same change here:

```suggestion
  @ExtraName("i") @ValueDescription("[default: false] - Skip the SSL certificate verification when communicating with the Codacy API")
```